packages/admin/src/error/errordialog/ErrorDialogContext.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from "react";
+
+import { ErrorDialogOptions } from "./ErrorDialog";
+
+export interface ErrorDialogContextProps {
+    showError: (options: ErrorDialogOptions) => void;
+}
+
+const ErrorDialogContext = React.createContext<ErrorDialogContextProps>(undefined!);
```

Comment:
please create a proper context default value, like
```
React.createContext<ErrorDialogContextProps>({
    showError: (options: ErrorDialogOptions) => {}
})
```

....I don't want to know what `undefined!` results in :)

Reply:
changed typo to `ErrorDialogContextProps | undefined`